queries.sql

```diff
@@ -16,4 +16,68 @@ SELECT * FROM animals WHERE name='Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon'
+WHERE name LIKE '%mon%';
+
+UPDATE animals SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+
+SELECT name, species FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+ DELETE FROM animals
+ WHERE date_of_birth > '01-01-2022';
+ SAVEPOINT SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK TO SP1;
+
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+
+SELECT count(*) FROM animals;
+
+SELECT count(*) FROM animals
+WHERE escape_attempts = 0;
+
+SELECT AVG(weight_kg)
+FROM animals;
+
+SELECT 
+  MAX(escape_attempts)
+FROM animals
+WHERE neutered = TRUE OR neutered = FASLE;
+
+SELECT   
+  MAX(weight_kg),
+  MIN(weight_kg)
+FROM animals
+WHERE species = 'digimon';
+
+SELECT   
+  MAX(weight_kg),
+  MIN(weight_kg)
+FROM animals
+WHERE species = 'pokemon';
+
+SELECT   
+  AVG(escape_attempts) ::numeric(10,2)
+FROM animals
+WHERE date_of_birth >= '1990-01-01' AND date_of_birth <= '2000-31-12';
```

Comment:
# [OPTIONAL] 
* I am marking this as optional because it wasn't mentioned by the previous CR but i strongly recommend you fix it as your queries won't work 
* Kindly correct this date format as it throws an error the correct format is `YYYY-MM-DD` instead of `YYYY-DD-MM` 
![image](https://user-images.githubusercontent.com/52497246/183858701-9860e044-32c2-416e-9191-0015e95b4761.png)